Framework/Algorithms/src/CreateTransmissionWorkspace2.cpp

```diff
@@ -147,6 +147,8 @@ void CreateTransmissionWorkspace2::exec() {
     stitch->setPropertyValue("StartOverlap", getPropertyValue("StartOverlap"));
     stitch->setPropertyValue("EndOverlap", getPropertyValue("EndOverlap"));
     stitch->setPropertyValue("Params", getPropertyValue("Params"));
+    stitch->setPropertyValue("ScaleRHSWorkspace",
```

Comment:
Personally I would use `setProperty`/`getProperty` though that has not been the general trend in this method to date. I would not use `setPropertyValue`/`getPropertyValue`. Type safety would be better, and no reason to serialize/deserialze to strings.